### DISCUSSION TITLE
/mnt/docker-aio-config is not a mountpoint

### DISCUSSION THREAD

**Discussion by IssueFindings** [nextcloud/all-in-one#3817] [Questions] [answered]
Hello,
It's a totally new install of NextCloud AIO through Docker on Ubuntu last version stable.
Based on the "official" command, I run this personal one below.
`docker run --init --sig-proxy=false --name nextcloud-aio-mastercontainer --restart always --publish 8081:8080 --env NEXTCLOUD_DATADIR="/home/USER/ncdata/data" --env NEXTCLOUD_MOUNT="/home/USER/ncdata/" --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-configconfig --volume /var/run/docker.sock:/var/run/docker.sock:ro nextcloud/all-in-one:latest`

i) I need to store NextCloud data on a specific directory located in /home/USER/ncdata/data, that the reason why I introduce `--env NEXTCLOUD_DATADIR="/home/USER/ncdata/data"`

ii) I need to redirect the output port to 8081, that the reason why I introduce `--publish 8081:8080`

iii) I don't touch `--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-configconfig` but I introduce `--env NEXTCLOUD_MOUNT="/home/USER/ncdata/"` to group every NextCloud data/configuration in /home/USER/ncdata/

I run Docker rootless without any issue with my other containers. I just add USER in the group DOCKER.

For me, it looks like good. Nevertheless I get this error message "/mnt/docker-aio-config is not a mountpoint. Cannot proceed!"
When I run `docker logs nextcloud-aio-mastercontainer` no more than "/mnt/docker-aio-config is not a mountpoint. Cannot proceed!"
When I run `docker inspect nextcloud-aio-mastercontainer` I still see that mountpoint is /mnt/docker-aio-configconfig and not /home/USER/ncdata/docker-aio-configconfig. Is it normal ?

Something is wrong, but I don't see what... I need help.

Have a nice day.

**Comment by szaimen** [comment, marked as answer]
> --volume nextcloud_aio_mastercontainer:/mnt/docker-aio-configconfig

This is your issue

**Reply by IssueFindings** [reply]
Many thanks for your answer.
So, I need to delete this option or rewrite it ? I tried `--volume nextcloud_aio_mastercontainer:/home/USER/ncdata/docker-aio-config` but that didn't work.
Have a nice day.

**Reply by szaimen** [reply]
You need to use `--volume nextcloud_aio_mastercontainer:/mnt/docker-aio-config`

**Reply by IssueFindings** [reply]
I'm confused... It's a basic mistake.
Many thanks for your help.
Have a nice day.
